docs/README.md

```diff
@@ -0,0 +1,161 @@
+## Run a full diracx demo
+
+This will allow you to run a demo setup.
+
+The code changes will be reflected in the demo.
+
+Requirement: docker, internet
+
+```bash
+# Clone the diracx repository
+git clone git@github.com:<yourfork>/diracx.git
+
+# Clone the diracx-chart repository
+git clone git@github.com:<yourfork>/diracx-charts.git
+
+# Run the
+diracx-charts/run_demo.sh diracx/
+```
```

Comment:
```suggestion

0. fork, in GitHub, the https://github.com/DIRACGrid/diracx/ and https://github.com/DIRACGrid/diracx-charts/ repositories.

```bash
git clone git@github.com:<yourfork>/diracx.git
cd diracx
git remote add upstream git@github.com:DIRACGrid/diracx.git
git fetch upstream

# Clone the diracx-chart repository
git clone git@github.com:<yourfork>/diracx-charts.git
cd diracx-charts
git remote add upstream git@github.com:DIRACGrid/diracx-charts.git
git fetch upstream


# Run the
diracx-charts/run_demo.sh diracx/
```
```

Reply:
I don't think this is necessary, it makes the instructions longer for no added value really. Plus, if they want to run a demo, they don't need to fork it.